src/modules/utils/structures/parcel.py

```diff
@@ -4,11 +4,12 @@ class Parcel:
     Target is a destination Node
     Default package volume for poc is 1, for easier UUID unpacking
     """
-    def __init__(self, uuid, target, volume, current_location, pickup_location=None, destination_location=None, type=None):
+    def __init__(self, uuid, target, volume, current_location, pickup_location=None, destination_location=None, type=None, country=None):
         self.volume = volume
         self.target = target
         self.uuid = uuid
         self.current_location = current_location
         #self.pickup_location = pickup_location
         #self.destination_location = destination_location
-        self.type = type
```

Comment:
remove self.pickup_location and destination_location if they are commented

Reply:
fixed